index.html

```diff
@@ -414,359 +342,358 @@ <h2>
-specification. It is understood that, in the same way that a web browser is not
-required to implement all known URI schemes, conformant software that works
-with <a>DIDs</a> is not required to implement all known <a>DID methods</a>.
-However, all implementations of a given <a>DID method</a> must be interoperable
-for that method.
-    </p>
-  </section>
-
-  </section>
-
-  <section class="informative">
-    <h1>
-Terminology
-    </h1>
-
-    <div data-include="terms.html" data-oninclude="restrictReferences">
-    </div>
-
-  </section>
-
-  <section>
-    <h1>Overall Architecture</h1>
-
-    <section>
-      <h2>DIDs</h2>
-    </section>
-
-    <section>
-      <h2>DID Registries</h2>
+            </td>
+          </tr>
+        </tbody>
+      </table>
     </section>
 
     <section>
-      <h2>DID Documents</h2>
+      <h2>
+Architecture Overview
+      </h2>
 
       <p>
-A <a>DID document</a> is the <a>resource</a> that is associated with a
-<a>decentralized identifier</a> (DID). <a>DID documents</a> typically express
-verification methods (such as public keys) and services that can be used to
-interact with a <a>DID controller</a>.
+This section provides a basic understanding of the major elements of DID
+architecture. Formal definitions of terms are provided in <a href="#terminology"></a>.
       </p>
 
-      <p>
-A <a>DID document</a> is serialized according to a particular syntax, as
-outlined in Section <a href="#core-representations"></a>). The <a>DID</a> itself
-is contained in the <code>id</code> property.
-      </p>
+      <dl>
+        <dt>
+DIDs and DID URLs
+        </dt>
+        <dd>
+A <a>DID</a>, or Decentralized Identifier, is a URI composed of three parts:
+the scheme "did:", a method identifier, and a unique identifier generated by
+the <a>DID method</a>. DIDs are resolvable to <a>DID documents</a>. A
+<a>DID URL</a> extends the syntax of a basic DID to incorporate other standard
+URI components (path, query, fragment) in order to locate a particular
+resource&mdash;for example, a public key inside a <a>DID document</a>, or a resource
+available external to the <a>DID document</a>.
+        </dd>
 
-      <p>
-The properties that can be present in a <a>DID document</a> are outlined in
-Section <a href="#did-documents"></a>.
-      </p>
+        <dt>
+DID Subjects
+        </dt>
+        <dd>
+The subject of a <a>DID</a> is, by definition, the entity identified by the
+<a>DID</a>. The <a>DID subject</a> may also be the <a>DID controller</a>.
+Anything can be the subject of a DID: person, group, organization, physical
+thing, logical thing, etc.
+        </dd>
 
-      <p>
-The properties present in a <a>DID document</a> can be updated according to
-the applicable operations outlined in Section <a href="#methods"></a>.
-      </p>
+        <dt>
+DID Controllers
+        </dt>
+        <dd>
+The <a>controller</a> of a <a>DID</a> is the entity (person or organization)
```

Comment:
```suggestion
The <a>controller</a> of a <a>DID</a> is the entity (person, organization, or autonomous software)
```

Reply:
Isn't all software autonomous? could just say "software"... may want "for example" as well as there are other examples of entities.

Reply:
redundant redundancy 🙃 

In my mind I was making the distinction between software that requires user interaction to complete a workflow vs software that doesn't require any interaction other than being started.